plugins/generator-1.20.4/neoforge-1.20.4/painting.definition.yaml

```diff
@@ -1,12 +1,12 @@
-global_templates:
-  - template: elementinits/paintings.java.ftl
-    name: "@SRCROOT/@BASEPACKAGEPATH/init/@JavaModNamePaintings.java"
-  - template: json/paintingtag.json.ftl
-    writer: json
-    name: "@RESROOT/data/minecraft/tags/painting_variant/placeable.json"
-
-localizationkeys:
-  - key: painting.@modid.@registryname.title
-    mapto: title
-  - key: painting.@modid.@registryname.author
+global_templates:

```

Comment:
Do you maybe know why github is showing those as changed?

Reply:
No idea why that happened, but it should be fixed now